### ISSUE TITLE
Feature request: continuous or interval-based recognition

### ISSUE THREAD

**Issue by daltux** [aleksey-saenko/MusicRecognizer#20]
It would be very nice to have an option to do continous recognition or, if more viable, to start it every minute (or some user setting).

**Comment by aleksey-saenko** [comment]
Hello, thanks for the suggestion.

This feature would be useful, and I have already thought about implementing it. It's possible to optimize the number of requests to the service: we can avoid sending during silence, we can make requests only after the end of the track (the app knows the track duration and the timestamp when the recognition occurred). However, the number of requests will still be large, and this is a problem.

AudD is not a free service and requires payment for requests. It is assumed that the developer buys a token and provides a paid application or monetizes it in another way, such as through advertising. I created this application not for commercial purposes, so the application relies on the user to obtain a token.

I think most users rely on current limit, which is around 3-5 recognitions per day (the new token has more requests, but only for 2 weeks). Therefore, as long as the app exists in this form, I don't think this feature will be in high demand.

I will leave this issue open, as perhaps a solution will be found someday.

**Comment by famewolf** [comment]
Could the app just store and display Info such as the date and time, track length, song signature, etc. In a history type page and then the user can come back later and look up actual ones that they want. When I'm in the car and I hear a song on the radio that I like, it would be nice if it could save that information and then I could come back later and specify which song I wanted to look up. This is more aimed at music that is ambient, I guess, than specific apps.

**Comment by catsgomoo** [comment]
I would also love a feature like this for when I'm listening to media that can't be scrobbled (Records, CDs, Tapes or what famewold described). From my understanding as well the API cost issue doesn't seem to be an issue anymore either.

Even if it's only active while the app is open that would greatly help my use cases

**Comment by famewolf** [comment]
Agreed.  Something like tasker could be used to auto start the app when in the car and kill the app when parked for my particular use case.

**Comment by kamik-hub** [comment]
Something like Ambient Music Mod, would be nice to have this feature
